domain-models-maven-plugin/src/test/resources/clients/TestClient.txt

```diff
@@ -107,10 +108,23 @@ public class TestResourceClient {
     }
 
     /**
-     * Service endpoint "unittests"+queryParams.toString()
-     * 
+     * Service endpoint unittests
+     * @deprecated Use method, that returns Future
+     *
      */
+    @Deprecated
```

Comment:
Neither Vert.x nor RMB deprecate `Handler<AsyncResult<>>` methods. We continue to support them.

When writing new code or changing existing code developers will likely use futurised methods for ease of use.

Reply:
I removed deprication